tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,11 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    assert not result
```

Comment:
If `result` was an empty list or any falsy value this `assert` statement would be true. How could be change this to specifically check for `False`.